Gifski/Gifski.swift

```diff
@@ -19,10 +19,10 @@ final class Gifski {
 				return "Failed to generate frame: \(error.localizedDescription)"
 			case let .addFrameFailed(error):
 				return "Failed to add frame, with underlying error: \(error.localizedDescription)"
-			case let .endAddingFramesFailed(error):
-				return "Failed to end adding frames, with underlying error: \(error.localizedDescription)"
-			case let .writeFailed(error):
-				return "Failed to write to output, with underlying error: \(error.localizedDescription)"
+			case let .startWritingFailed(error):
+				return "Failed to start writing, with underlying error: \(error.localizedDescription)"
+			case let .finishWritingFailed(error):
+				return "Failed to finish writing, with underlying error: \(error.localizedDescription)"
```

Comment:
@kornelski Are these error names correct?

Reply:
The user-facing message doesn't need to distinguish between start/finish explicitly. The `gifski_finish()` call will report any error during writing.

Reply:
Like this https://github.com/sindresorhus/Gifski/pull/90/commits/ec5905990312aceb079fcf522d0e603d821cf8d6 ?